### ISSUE TITLE
File picker doesn't show when changing a contact picture

### ISSUE THREAD

**Issue by heydrickx** [nextcloud/contacts#3770]
### Describe the bug

After clicking on the picture and also on "Select from files", a window pops up but the list of the files is not loaded, even after waiting a long time.
Tested on Firefox and Chrome.
Nothing is reported in the logs.
The problem existed on Contacts app V5.5.0, and remained with V5.5.1
I can't really tell if I had made any change before the problem happened. It might be linked to the upgrade to Nextcloud V28.

### Steps to reproduce

1. Go to Contacts app
2. Click on a contact (the issue occurs whether the contact already has a picture or not) and click on 'Edit'; or create a new contact
3. Click on the "picture logo" on top of the contact picture
4. Select 'Select from file' from the menu
5. See error

(_be advised that I've translated the labels from french... they may not be exactly the same in the english version_)

### Expected behavior

Loading the list of files and folders enabling to select a picture.

### Actual behavior

The window opens, but the list of files is not displayed. It looks like it is loading but nothing never happens
![nc_contacts](https://github.com/nextcloud/contacts/assets/33583616/97ee1bb2-8ee8-4664-9f78-f01dde23825c)
 (see picture).

### Contact version

5.5.0

### Operating system

Debian 12

### PHP engine version

Other

### Web server

Apache (supported)

### Database

MariaDB

### Additional info

PHP version 8.3.0

**Comment by ChristophWurst** [comment]
could you check the browser console for errors and logs?

**Comment by heydrickx** [comment]
> could you check the browser console for errors and logs?

After clicking on "Select from file", I see the below error (404 Not Found) in the console (Firefox).


```
<html><body>
<!--StartFragment--><span class="message-body-wrapper"><div class="url-preview"><div class="properties-view"><div class="tree-container">
https://_myserver_/remote.php/dav/files/_myuser_/
--


</span><!--EndFragment-->
</body>
_</html>https://_myserver_/remote.php/dav/files/myuser/_
Status
404
Not Found
VersionHTTP/1.1
Transferred553 B (272 B size)
Referrer Policyno-referrer
Request PriorityHighest
DNS ResolutionSystem

	
Connection
	Keep-Alive
Content-Length
	272
Content-Type
	text/html; charset=iso-8859-1
Date
	Thu, 04 Jan 2024 13:54:14 GMT
Keep-Alive
	timeout=5, max=100
Server
	Apache/2.4.57 (Debian)
Strict-Transport-Security
	max-age=15552000; includeSubDomains
	
Accept
	text/plain,application/xml
Accept-Encoding
	gzip, deflate, br
Accept-Language
	en-US,en;q=0.5
Connection
	keep-alive
Content-Length
	598
Content-Type
	text/plain;charset=UTF-8
Depth
	1
DNT
	1
Host
	_myserver_
Origin
	https://_myserver_/
requesttoken
	t3L9AOCbp7pI4XWuHC3oH5YQXHX+FD/Jld5VjVzXWmw=:1D3POafiwes+sj7+bXeEWvh9BQKKQXKc7+c3yBCTbhU=
Sec-Fetch-Dest
	empty
Sec-Fetch-Mode
	cors
Sec-Fetch-Site
	same-origin
Sec-GPC
	1
User-Agent
	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:121.0) Gecko/20100101 Firefox/121.0
X-Requested-With
	XMLHttpRequest
Error: Invalid response: 404 Not Found
    pt index.js:2
    mt index.js:2
    Xt index.js:2
    promise callback*Kt index.js:2
    Xt index.js:2
    Xt index.js:2
    getDirectoryContents index.js:2
    a FilePicker-0bd54f66.mjs:835
    setup FilePicker-0bd54f66.mjs:927
    VueJS 13
    pick index.mjs:30
    pick index.mjs:56
    pick index.mjs:55
    filepicker dialogs.js:360
    value index.mjs:86
    value index.mjs:85
    n ContactDetailsAvatar.vue:479
    d ContactDetailsAvatar.vue:110
    w ContactDetailsAvatar.vue:110
    T ContactDetailsAvatar.vue:110
    bI ContactDetailsAvatar.vue:110
    o ContactDetailsAvatar.vue:110
    yI ContactDetailsAvatar.vue:110
    yI ContactDetailsAvatar.vue:110
    selectFilePicker ContactDetailsAvatar.vue:464
    VueJS 4
    onClick actionText-60ff01d1.mjs:114
    VueJS 41
[vue.runtime.esm.js:3049:16](webpack:///nextcloud/node_modules/vue/dist/vue.runtime.esm.js)
Error: Invalid response: 404 Not Found
    pt index.js:2
    mt index.js:2
    Xt index.js:2
    promise callback*Kt index.js:2
    Xt index.js:2
    Xt index.js:2
    getDirectoryContents index.js:2
    a FilePicker-0bd54f66.mjs:835
    <anonymous> FilePicker-0bd54f66.mjs:843
    VueJS 13
    pick index.mjs:30
    pick index.mjs:56
    pick index.mjs:55
    filepicker dialogs.js:360
    value index.mjs:86
    value index.mjs:85
    n ContactDetailsAvatar.vue:479
    d ContactDetailsAvatar.vue:110
    w ContactDetailsAvatar.vue:110
    T ContactDetailsAvatar.vue:110
    bI ContactDetailsAvatar.vue:110
    o ContactDetailsAvatar.vue:110
    yI ContactDetailsAvatar.vue:110
    yI ContactDetailsAvatar.vue:110
    selectFilePicker ContactDetailsAvatar.vue:464
    VueJS 4
    onClick actionText-60ff01d1.mjs:114
    VueJS 41
```

**Comment by ChristophWurst** [comment]
What version of Nextcloud is that?

**Comment by heydrickx** [comment]
> What version of Nextcloud is that?

It is Nextcloud version 28.0.1

**Comment by heydrickx** [comment]
Hello,

the issue has been **resolved** with the update to version 28.0.2
